ckanext/canada/migration/canada_public/versions/0ef791477ff0_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: 0ef791477ff0
+Revises:
+Create Date: 2024-10-18 18:20:50.040861
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0ef791477ff0'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    from ckanext.canada.model import PackageSync
```

Comment:
we can't reference the model from a migration because it will change over time. Instead either have alembic generate the migration for you ( https://github.com/ckan/ckan/pull/8239 ) , or copy and modify an example from another extension.